pkg/cnab/config-adapter/adapter.go

```diff
@@ -20,14 +20,22 @@ const SchemaVersion = "v1.0.0"
 type ManifestConverter struct {
 	*context.Context
 	Manifest     *manifest.Manifest
+	ManifestPath string
 	ImageDigests map[string]string
 	Mixins       []mixin.Metadata
 }
 
-func NewManifestConverter(cxt *context.Context, manifest *manifest.Manifest, imageDigests map[string]string, mixins []mixin.Metadata) *ManifestConverter {
+func NewManifestConverter(
+	cxt *context.Context,
+	manifest *manifest.Manifest,
+	manifestPath string,
+	imageDigests map[string]string,
+	mixins []mixin.Metadata,
+) *ManifestConverter {
 	return &ManifestConverter{
 		Context:      cxt,
 		Manifest:     manifest,
+		ManifestPath: manifestPath,
```

Comment:
I'm curious if you looked into using `manifest.ManifestPath` instead of passing it around as an extra argument? Was the value not reliable?

If we could rely on that value, then we could revert a lot of the changes in this PR.

Reply:
Good thinking!  Forgot all about that field.  Removed the extra arg.